### PR TITLE
add arbitrum error already known

### PR DESCRIPTION
**Pull request by poopoothegorilla**
One of the RPCs on Arbitrum Sepolia returns 400 status code on TransactionAlreadyKnown error, that breaks our error classification and causes MultiNode to log critical error due to contradictions in results (one of the RPCs returns success and TransactionAlreadyKnown is classified as Unknown)

### CODE REVIEW

File: core/chains/evm/client/errors.go [hunk 1]
```diff
@@ -88,7 +88,7 @@ var geth = ClientErrors{
 	NonceTooLow:                       regexp.MustCompile(`(: |^)nonce too low$`),
 	NonceTooHigh:                      regexp.MustCompile(`(: |^)nonce too high$`),
 	ReplacementTransactionUnderpriced: regexp.MustCompile(`(: |^)replacement transaction underpriced$`),
-	TransactionAlreadyInMempool:       regexp.MustCompile(`(: |^)(?i)(known transaction|already known)`),
+	TransactionAlreadyInMempool:       regexp.MustCompile(`(?i)(known transaction|already known)`),
```
Comment (jmank88) [inline core/chains/evm/client/errors.go:91]: Why here instead of `arbitrum` on line 134?

Comment (simsonraj) [reply core/chains/evm/client/errors.go:91]: Tenderly could be doing this for other errors too, which we may not have encountered yet?

Comment (dimriou) [reply core/chains/evm/client/errors.go:91]: Many people use this file as an index, so I'd also suggest to move this under `arbitrum`. If it's being used by others but we don't know it's ok, but at least let's be consistent with what we know so far.

**Review by jmank88 — APPROVED** [review]

**Review by simsonraj — APPROVED** [review]

**Review by prashantkumar1982 — COMMENTED** [review]
Did we get an answer from Tenderly if this was indeed an intended change?
Last I heard, they acknowledged they may have mis-categorized this under 400 instead of 200.
If they agree and fix their side, do we still want to go ahead with this fix?

**Pull request closed without merging**